common/core/desktop/src/kmx/kmx_processor.cpp

```diff
@@ -173,6 +179,9 @@ KMX_BOOL KMX_Processor::ProcessGroup(LPGROUP gp, KMX_BOOL *pOutputKeystroke)
   {
     DebugLog("Aborting output: m_state.LoopTimes exceeded.");
     m_state.StopOutput = TRUE;
+    if(m_debug_items) {
+      m_debug_items->push_group_exit(gp, KM_KBP_DEBUG_FLAG_RECURSIVE_OVERFLOW);
+    }
```

Comment:
Corresponds to:
https://github.com/keymanapp/keyman/blob/75e47c1ccc36e1698bafa8178acf8140f88e4e2b/windows/src/engine/keyman32/kmprocess.cpp#L252